src/components/safe-messages/SingleMsg/SingleMsg.test.tsx

```diff
@@ -0,0 +1,88 @@
+import { extendedSafeInfoBuilder } from '@/tests/builders/safe'
+import { fireEvent, render, waitFor } from '@/tests/test-utils'
+import * as useSafeInfo from '@/hooks/useSafeInfo'
+import * as syncSafeMessageSigner from '@/hooks/messages/useSyncSafeMessageSigner'
+
+import { SafeMessageListItemType, SafeMessageStatus, type SafeMessage } from '@safe-global/safe-gateway-typescript-sdk'
+import SingleMsg from '.'
+
+const SAFE_ADDRESS = '0x87a57cBf742CC1Fc702D0E9BF595b1E056693e2f'
+const SAFE_MESSAGE_HASH = '0x32d898a35478c1c2063f76841cd99388e28ddac217e17bf787e2307ba8bf860b'
+
+// Minimum mock to render <SingleMsg />
+const msgDetails = {
```

Comment:
Nice to have:
Ideally we create a `safeMsgBuilder` for this. Similar to our `safeInfoBuilder` and `safeTxBuilder`. Really handy to mock quickly.